src/components/search/data/constants.js

```diff
@@ -1,34 +1,48 @@
-export const ALGOLIA_INDEX_NAME = 'enterprise_catalog_new';
+export const ALGOLIA_INDEX_NAME = 'enterprise_new';
+
+export const FREE_ALL_ATTRIBUTE = 'showAll';
+export const SHOW_ALL_NAME = 'showAll';
+export const FREE_TO_ME_NAME = 'freeToMe';
 
 export const SEARCH_FACET_FILTERS = [
   {
     attribute: 'subjects',
     title: 'Subject',
+    name: 'subjects',
```

Comment:
Ended up using `attribute` instead of name, so remove these if they didn't end up being used elsewhere.